### PR TITLE
Add several spelling corrections + fix some pre-existing

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11525,6 +11526,8 @@ finess->finesse
 finialization->finalization
 finializing->finalizing
 finilizes->finalizes
+finisch->finish, finish,
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt:11529]: The word `finish` is mentioned twice. Did you mean `Finnish` in one of the cases (language in Finland)?

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:11529]: This was caught by the tests (although the error message isn't very clear):
https://github.com/codespell-project/codespell/blob/bd79d997e62b87ef9131f5539e539f853b072ec5/codespell_lib/tests/test_dictionary.py#L118

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18951,6 +18955,7 @@ practicly->practically
 practicval->practical
 practioner->practitioner
 practioners->practitioners
+practise->practice
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:18958]: Not a typo:
https://en.wikipedia.org/wiki/Practice_(learning_method)

Possibly could go in the BE to AmE dictionary:
> In English, practice is the noun and practise is the verb, but in American-English, the historically erroneous confusion of cases in America means it is now common for practice to be used both as a noun and a verb

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -19874,7 +19879,7 @@ quroum->quorum
 qutie->quite, quiet,
 rabinnical->rabbinical
 racaus->raucous
-ractise->practise
+ractise->practice
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19882]: Please undo this change as above, although you could add `ractice->practice` too

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -25395,7 +25400,9 @@ tupples->tuples
 ture->true
 turle->turtle
 turly->truly
-turnk->turnkey, trunk,
+turnk->trunk, turnkey,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25403]: Also suggest turn.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:11529]: Message improved in #1493

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:11529]: This is in now, so would you mind doing a Git update first @luzpaz before you fix this, so I can confirm it worked fine in Travis before you commit the fix?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @peternewman result: https://travis-ci.org/github/codespell-project/codespell/jobs/698171480#L738  
```
E           AssertionError: 
E           error finisch: corrections "finish, finish," are not (lower-case) unique
```

**Comment by luzpaz** [comment]
Not sure what the problem is: https://travis-ci.org/github/codespell-project/codespell/jobs/698172331#L656

**Comment by peternewman** [comment]
> Not sure what the problem is: https://travis-ci.org/github/codespell-project/codespell/jobs/698172331#L656

You've added:
`ractise->practice`

It's complaining that `ractise` isn't a valid word in Aspell, which doesn't surprise me. This correction doesn't need to be in this dictionary as you've already added `practise->practice` as I suggested.

**Comment by luzpaz** [comment]
I'm confused since they look like separate typos to me? does aspell recognize `ractise` as a typo ?

**Comment by peternewman** [comment]
> I'm confused since they look like separate typos to me? does aspell recognize `ractise` as a typo ?

The key thing is aspell doesn't recognise `ractise` as a valid word.

The GB to US dictionary is currently expected to consist of valid words on both sides (this slightly falls down as some more niche words, such as #1562, aren't in the dictionary). They should just be things like `colour->color` not `coolour->color`.

**Comment by luzpaz** [comment]
Can we have a dictionary of non-recognizable aspell words then ?

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 5]
```diff
@@ -185,6 +185,7 @@ publicise->publicize
 publicised->publicized
 publicises->publicizes
 publicising->publicizing
+ractise->practice
```
Comment (larsoner) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:188]: I don't think `ractise` is a real word, even in British English?

Comment (luzpaz) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:188]: `practise` is a real word and is the variant of `practice`but the British and Australian spelling of it. `ractise` is just a typo of `practise`

Comment (sebweb3r) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:188]: yeah, but the dictionary_en-GB_to_en-US.txt wants  
valid Be->valid AE

you want to add the typo to the regular dictionary.
